sysutils/yabai/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            koekeishiya yabai 3.3.0 v
+github.tarball_from     releases
+
+categories              sysutils
+platforms               darwin
+maintainers             {@thekevjames thekev.in:macports} openmaintainer
+license                 MIT
+description             A tiling window manager for macOS based on binary space partitioning
+long_description        yabai is a window management utility that is designed to work as an extension to the built-in window manager of macOS.
+distname                ${github.project}-v${github.version}
+
+supported_archs         x86_64
+checksums               rmd160  85d63f56aa1ef53b300ac3228236168305dbad50 \
+                        sha256  8ccb7ba4e3522fa6f52545a9d5ac0c5efb7b0720ed9c2986605022a01c9938ce \
+                        size    249829
+
+set yabai.plist         ${workpath}/org.macports.yabai.plist
+
+post-patch {
+    # Fill in startup plist template
+    copy -- ${filespath}/org.macports.yabai.plist.in ${yabai.plist}
+    reinplace "s|@PREFIX@|${prefix}|g" ${yabai.plist}
+}
+
+use_configure           no
+
+build {}
+
+destroot {
+    # Copy pre-built binary
```

Comment:
update comment; you still need to do this, but we copy the file we just compiled (i.e., just get rid of the `pre-built` here)